onnx2kerastl/pooling_layers.py

```diff
@@ -154,14 +154,16 @@ def convert_global_avg_pool(node, params, layers, lambda_func, node_name, keras_
     :param keras_name: resulting layer name
     :return: None
     """
-
     input_0 = ensure_tf_type(layers[node.input[0]], layers[list(layers)[0]], name="%s_const" % keras_name)
-
-    global_pool = keras.layers.GlobalAveragePooling2D(data_format='channels_first', name=keras_name)
+    tensor_dim = len(input_0.shape)
+    if tensor_dim == 3:
+        global_pool = keras.layers.GlobalAveragePooling1D(data_format='channels_first', name=keras_name)
+    else:
```

Comment:
I Recommend adding here `if tensor_dim == 4:` for this condition and another `else` after that throws an Exception explaining what happened

Reply:
OK